src/main/kotlin/racingcar/view/InputView.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.view
+
+import camp.nextstep.edu.missionutils.Console
+
+object InputView {
+    fun askForCarNames(): String {
+        println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)")
+        return Console.readLine()
```

Comment:
개인적으로 전 이렇게 변수 선언을 한번 거친 후 리턴하는 방법을 통하면 좀 더 명확해지는거 같습니다!!😃
```suggestion
        val carNames = Console.readLine()
        return carNames
```